packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -65,6 +65,43 @@ import 'theme.dart';
 /// ```
 ///
 ///{@end-tool}
+///
```

Comment:
This should probably be in the doc comment for the .builder() constructor as it is specific to that function.

Reply:
done moved this example to .builder constructor